sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -899,16 +899,16 @@ static void validateMultipageReceiptData(List<RecognizedForm> recognizedReceipts
             .getValue().asString());
         assertEquals(EXPECTED_MULTIPAGE_PHONE_NUMBER_VALUE, receiptPage1Fields.get("MerchantPhoneNumber")
             .getValue().asPhoneNumber());
-        assertNotNull(receiptPage1Fields.get("Total").getValue().asFloat());
+        // service issue total returned as null?
```

Comment:
do you think creating a github issue and putting it here will help to keep track of this?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/17661